K2Bridge.Tests.UnitTests/KustoDAL/KustoDataAccessTests.cs

```diff
@@ -178,8 +338,8 @@ public async Task GetFieldCaps_WithValidFunction_ReturnFieldCaps()
 
             mockQueryExecutor.Verify(exec => exec.ExecuteQueryAsync(
                 It.Is<QueryData>(d =>
-                d.IndexName == "testIndexName"
-                && d.QueryCommandText == "testIndexName | getschema | project ColumnName, ColumnType=DataType"), It.IsAny<RequestContext>()));
+                    d.IndexName == "testIndexName"
+                    && d.QueryCommandText == "testIndexName | getschema | project ColumnName, ColumnType=DataType"), It.IsAny<RequestContext>()));
```

Comment:
did we go with build schema?

Reply:
This is unrelated to this PR (the change is only in indentation, maybe I should change my IDE's settings and revert the indentation back)

We use `getschema` to get the regular fields of the table.

`buildschema` only works with dynamic columns so we use it only for that case